homeassistant/components/feedreader/config_flow.py

```diff
@@ -116,11 +96,6 @@ async def async_step_user(
             options={CONF_MAX_ENTRIES: self._max_entries or DEFAULT_MAX_ENTRIES},
```

Comment:
You can drop `_max_entries` class attribute
```suggestion
            options={CONF_MAX_ENTRIES: DEFAULT_MAX_ENTRIES},
```

Reply:
Thanks!

Reply:
You still need to remove the class attribute

Reply:
done